### DISCUSSION TITLE
South Africa VoLTE works with Rain carrier

### DISCUSSION THREAD

**Discussion by JacoBriers** [kyujin-cho/pixel-volte-patch#181] [Carrier-specific Experiences]
Many thanks! Just tested VoLTE on the Rain carrier in South Africa and worked with no issues.

**Comment by fillwithjoy1** [comment]
Great news. I'll add this to the documentation. Can you verify if VoWiFi works and send a screenshot of your main APN (Access Point Name). I'm specifically looking for MCC and MNC.
